crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -337,6 +338,8 @@ impl EcmascriptChunkItem for ModuleChunkItem {
                 }
                 program.visit_mut_with(&mut swc_core::ecma::transforms::base::hygiene::hygiene());
                 program.visit_mut_with(&mut swc_core::ecma::transforms::base::fixer::fixer(None));
+
+                remove_shebang(&mut program);
```

Comment:
Please add a comment explaining _why_ we're removing the shebang, either here or in the function itself.

Reply:
done